api-connection.html

```diff
@@ -102,6 +103,109 @@ <h2 id="function_newConnection">new Connection(config)</h2>
         <code>azure-active-directory-password</code>, <code>azure-active-directory-access-token</code>, 
         <code>azure-active-directory-msi-vm</code>, <code>azure-active-directory-msi-app-service</code>, 
         or <code>azure-active-directory-service-principal-secret</code>
+        <ul>
+          <li><code>azure-active-directory-password</code></li>
+{% highlight javascript %}
+"authentication": {
+  "type":'azure-active-directory-password',
+  ...,
+  "options": {
+    "userName": value,
+    "password": value,
+    "domain": value (Optional)
+  }
+}           
+{% endhighlight %}
+          <ul>
+            <li>username (string): A user needs to provide the `userName` associated to their account.</li>
+            <li>password (string): A user needs to provide the `password` associated to their account.</li>
+            <li>domain (string): Optional parameter for specific Azure tenant ID.</li>
+          </ul>
+          <li><code>azure-active-directory-access-token</code></li>
+{% highlight javascript %}
+"authentication": {
+  "type":'azure-active-directory-access-token',
+  ...,
+  "options": {
+    "token": value
+  }
+}         
+{% endhighlight %}
+          <ul>
+            <li>token (string): A user needs to provide the `token` which they retrieved elsewhere to forming the connection.</li>
+            <p>
+              <h4>Best Practices</h4>
+              Tedious supports an authentication type: <code>azure-active-directory-password</code>. 
+              This is the recommended method because Tedious will handle the process of retrieving the Azure token 
+              internally instead of requiring the user to manually retrieve the token as shown below.
+            </p>
+            <p>
+              <h4>How to get the token</h4>
+              There are different approaches if a user actually in need of retriving a token. 
+              The detials can be found from microsoft side documentations. There are some examples:
```

Comment:
```suggestion
              There are different approaches if a user actually in need of retriving a token. 
              The detials can be found from microsoft side documentations. There are some examples:
```
              There are different approaches if a user actually needs to retrieve a token manually.
              The details can be found in Microsoft documentation. Here are some examples: